src/components/organisms/Dashboard/Tableview/NoResourcesFound.tsx

```diff
@@ -0,0 +1,47 @@
+import {useMemo} from 'react';
+
+import {Image} from 'antd';
+
+import styled from 'styled-components';
+
+import {activeProjectSelector} from '@redux/appConfig';
+import {useAppSelector} from '@redux/hooks';
+
+import NoDataFound from '@assets/NoDataFound.svg';
+
+const NoResourcesFound: React.FC = () => {
+  const activeProject = useAppSelector(activeProjectSelector);
+
+  const quickTipText = useMemo(() => {
+    if (activeProject) {
+      return 'Quick tip: Create resources of any kind from scratch on your local project. You can then deploy them to your cluster.';
+    }
+
+    return 'Quick tip: Start a local project and create resources of any kind from scratch. You can then deploy them to your cluster.';
+  }, [activeProject]);
+
+  return (
+    <Container>
+      <Image src={NoDataFound} width={100} height={100} />
+      <NotFoundText>No resources found</NotFoundText>
```

Comment:
could we be more elaborate? i.e. 
- "No <resourceKind> resources found in namespace <namespace>" if a namespace is selected"
- "No <resourceKind> resources found in cluster" if no namespace is selected
?